appengine/Rakefile

```diff
@@ -0,0 +1,19 @@
+$VERBOSE = true
+
+task :default => [:test]
+
+task :build do
+  ::Dir.chdir(::File.dirname(__FILE__)) do
+    puts("docker build -t appengine-ruby-base .")
```

Comment:
rake has `sh` to do a puts and system, and if you don't provide it a block, i believe it calls fail as expected.